tests/test_autocomplete.py

```diff
@@ -27,63 +27,40 @@ def read_fixture(sPath):
 
 @pytest.fixture
 def mocked_responses():
-    with responses.RequestsMock(assert_all_requests_are_fired=False) as rsps:
+    with respx.mock as rsps:
         yield rsps
 
 
 @pytest.fixture
 def mock_NLU(mocked_responses):
     with override_settings({"NLU_TOKENIZER_URL": NLU_URL, "NLU_CLASSIFIER_URL": CLASSIF_URL}):
-        mocked_responses.add(
-            responses.POST, f"{NLU_URL}", json=FIXTURE_TOKENIZER, status=200,
-        )
-        """
-        Two queries to the classifier API are required here,
-        as a consequence the order of the two queries below is important
-        """
-        mocked_responses.add(
-            responses.POST, f"{CLASSIF_URL}", json=FIXTURE_CLASSIF_pharmacy, status=200,
-        )
+        respx.post(NLU_URL, content=FIXTURE_TOKENIZER)
+        respx.post(CLASSIF_URL, content=FIXTURE_CLASSIF_pharmacy)
         yield mocked_responses
```

Comment:
As `mocked_responses` is not used anymore, isn't there a way to use respx's own decorator to avoid adding it?